<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CryptoApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="fileLabel">
      <property name="text">
       <string>Fichier choisi : Aucun</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="browseButton">
      <property name="text">
       <string>Parcourir...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="keyLineEdit">
      <property name="placeholderText">
       <string>Entrez la clé de chiffrement</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="algorithmComboBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>AES</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blowfish</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RSA</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="encryptButton">
        <property name="text">
         <string>Chiffrer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="decryptButton">
        <property name="text">
         <string>Déchiffrer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="hashButton">
        <property name="text">
         <string>Hasher</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="hashTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Les résultats des hachages MD5 et SHA-256 apparaîtront ici...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
